### ISSUE TITLE
A11Y_VSTSFeature1575027_Pipelines_RecentTabFolderView_Screenreader:Nvda does not announce the control type for "switch to list view" and "new folder" button.

### ISSUE THREAD

**Issue by fsteffi** [nvaccess/nvda#10249]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

**GitHubTags:** #A11yMAS; #A11yTCS; #MAS1.3.1; #NVDA; #Chrome; #A11Y_VSTSFeature1575027; #DesktopWeb; #Win10; #AzDev-Team Foundation Service;

**Environment Details:**
Application Name: VSTS Feature 1575027
Windows Version: Windows10
Browser Version: Chrome 76.0.3809.100 
Application URL: https://acctesting.visualstudio.com
NVDA Version:2019.2

**Steps to Reproduce:**
Navigate directly to this URL- https://acctesting.visualstudio.com/a11ytest/_build
or follow below steps:
1. Launch application using https://acctesting.visualstudio.com
2. Turn on multi-stage pipelines preview feature.​
3. Select any project.​
4. Navigate to pipelines hub & select pipeline.​
5. Navigate till Switch to folder view button & hit enter.​
6. Launch screen reader as nvda and navigate to "new folder" button.

**Actual:**
On navigating to new folder button nvda just announces it name "new folder 1 of 1". Similarly  for  "switch to list view" .This issue is not observed in edge+narrator configuration.

**Expected:**
On navigating to new folder button nvda should announce its name "new folder 1 of 1" as well as the control as "menu item" . Similarly  for  switch to list view". 

**User Impact:**
Screen reader users will not know if its a control or some data , also the action to take will not be known.

**Attachment Reference:**
![1590272_A11Y_VSTSFeature1575027_Pipelines_RecentTabFolderView_Screenreader](https://user-images.githubusercontent.com/24580570/65223888-b9b94600-dadf-11e9-8d41-43821abe941f.png)

**Comment by Adriani90** [comment]
@fsteffi I am testing with NVDA version Version: alpha-18699,0e4bb4ad 
on Chrome 77.0.3865.90 and I cannot reproduce this issue. Could you please test with the last alpha and see if it's still happening? Here you can download the last alpha versions of NVDA:
https://www.nvaccess.org/files/nvda/snapshots/

**Comment by fsteffi** [comment]
@Adriani90 the issue still reproduces with nvda version: alpha-18699,0e4bb4ad.
Below is the image of the issue 
![Image](https://user-images.githubusercontent.com/24580570/65322491-a0d69080-dbc4-11e9-9906-dea098ec17ae.png)

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
